hooks/terraform_docs.sh

```
@@ -240,10 +240,19 @@ function terraform_docs {
       [[ ! $have_marker ]] && popd > /dev/null && continue
     fi
-    local config_options
-    [[ $have_config_flag == true ]] && config_options="--config=$config_file"
-    # shellcheck disable=SC2086
-    terraform-docs --output-mode="$output_mode" --output-file="$output_file" $tf_docs_formatter "$config_options" $args ./ > /dev/null
+
+    local tfdocs_cmd=(
+      terraform-docs
+        --output-mode="$output_mode"
+        --output-file="$output_file"
+        $tf_docs_formatter
```

Shellcheck (0.10.0)

[warning] Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a.

(SC2206)